cmd/query/app/querysvc/adjuster/adjuster.go

```diff
@@ -9,20 +9,13 @@ import (
 	"go.opentelemetry.io/collector/pdata/ptrace"
 )
 
-// Adjuster defines an interface for modifying a trace object.
-// It returns the adjusted trace object, which is also updated in place.
-// If the adjuster encounters an issue that prevents it from applying
-// modifications, it should return the original trace object along with an error.
+// Adjuster is an interface for modifying a trace object in place.
+// If an issue is encountered that prevents modifications, an error should be returned.
+// The original trace object is modified directly and not returned.
+// The caller must ensure that all spans in the ptrace.Traces argument
+// belong to the same trace and represent the complete trace.
 type Adjuster interface {
-	Adjust(ptrace.Traces) (ptrace.Traces, error)
-}
-
-// Func is a type alias that wraps a function and makes an Adjuster from it.
-type Func func(traces ptrace.Traces) (ptrace.Traces, error)
```

Comment:
in case of simple adapters that can be coded as a single function it may be useful to keep the `Func`, but if nothing uses it then it's ok to delete.

Reply:
@yurishkuro Its not being used anywhere right now so I removed it. Can always add it back in later if its needed.